core/src/main/java/com/hivemq/testcontainer/core/HiveMQTestContainerCore.java

```diff
@@ -108,6 +109,14 @@ public HiveMQTestContainerCore(final @NotNull String image, final @NotNull Strin
         });
     }
 
+    @Override
+    public void start() {
+        super.start();
+        if (controlCenterEnabled) {
+            logger.info("The HiveMQ Control Center is reachable under: http://localhost:{}", getMappedPort(CONTROL_CENTER_PORT));
```

Comment:
if the user provides a custom config with a tls listener for the CC, then this log might be wrong, but I think it is fine for the default.